engine/src/main/java/org/terasology/engine/config/flexible/AutoConfigTypeHandler.java

```diff
@@ -43,7 +43,7 @@ protected PersistedData serializeNonNull(AutoConfig value, PersistedDataSerializ
                     if (typeHandler.isPresent()) {
                         fields.put(field.getName(), typeHandler.get().serialize(setting.get(), serializer));
                     } else {
-                        logger.error("Cannot serialize type [{}]", setting.getValueType());
+                        logger.atError().addArgument(() -> setting.getValueType()).log("Cannot serialize type [{}]");
```

Comment:
Simple getter method so impact neglectable
```suggestion
                        logger.error("Cannot serialize type [{}]", setting.getValueType()); //NOPMD
```